### ISSUE TITLE
Unable to load database names

### ISSUE THREAD

**Issue by fresheneesz** [paralect/robomongo#1011]
After upgrading to mongo 3.0, I'm getting the error:

```
Cannot connect to MongoDB (localhost:27017), error: Unable to load database names.
```

I can connect via the direct mongo shell from the console, but robo mongo can't connect. I've been able to connect before I upgraded. In fact, I was able to connect before I changed my config to YAML. My configuration is:

```
storage:
   dbPath: "/home/vagrant/backend/mongodata"
   engine: wiredTiger

systemLog:
   destination: file
   path: "/home/vagrant/backend/log/mongo.log"
   logAppend: true

net:
   bindIp: 127.0.0.1
   port: 27017
```

I'm using Robomongo 0.8.5

**Comment by fresheneesz** [comment]
Ok, it seems 0.8.5 doesn't support mongo 3 and 0.9.0 just came out: https://github.com/paralect/robomongo/issues/771

**Comment by fresheneesz** [comment]
0.9.0 has the same issue : /

**Comment by anorsich** [comment]
@fresheneesz It looks like your MongoDB instance is running inside vagrant. Are you sure that you've published port (27017) from guest to host? Are you connecting to the MongoDB via shell from the host machine or from vagrant VM (`vagrant ssh` than `mongo`)?

Robomongo 0.8.5 was able to connect to the MongoDB 3+, but collections were empty. I feel like the problem not in Robomongo.

**Comment by fresheneesz** [comment]
Like I said, this was working before I changed my config (in an attempt to upgrade my data to wired tiger), so I know the port is published in a way i can access from my host machine. The shell I connected to was on the guest side tho. My previous config was:

```
dbpath = /home/vagrant/backend/mongodata
logpath = /home/vagrant/backend/log/mongo.log
logappend = true
bind_ip = 127.0.0.1
```

I hear that the error "Unable to load database names" happens if you aren't authing properly. But I never had any auth, and my new config doesn't specify any so.. i'd assume there's still no auth required. Any idea what might be wrong with my config?
